Bottom: 528d1d39544763c39bb5496fefb186f874eb407b
Top:    d370dafbe5edd86422437837169ef245b12f98ee
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-25 14:08:57 +0100

Change scroll-wheel bindings to match more GTK applications









---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index e9469e9..7b0769d 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -25,18 +25,18 @@ Mouse =
     up-shift-ctrl = { Mode(Notify) Mode(Restore) Display(ToggleRubberbandMode) }
   }
   Up = {
-    Zoom(0.8)
-    shift = Scroll(up)
-    ctrl = Scroll(left)
+    Scroll(up)
+    shift = Scroll(left)
+    ctrl = Zoom(0.8)
   }
   Down = {
-    Zoom(1.25)
-    shift = Scroll(down)
-    ctrl = Scroll(right)
+    Scroll(down)
+    shift = Scroll(right)
+    ctrl = Zoom(1.25)
   }
 # If you want zoom to center, do this instead.
-  #Up = { {Zoom(0.8) Center()} }
-  #Down = { {Zoom(1.25) Center()} }
+  #Up = { ctrl = {Zoom(0.8) Center()} }
+  #Down = { ctrl = {Zoom(1.25) Center()} }
 }
 
 MainMenu =
